g4f/Provider/selenium/MyShell.py

```diff
@@ -2,10 +2,10 @@
 
 import time, json
 
-from ..typing import CreateResult, Messages
-from .base_provider import AbstractProvider
-from .helper import format_prompt
-from ..webdriver import WebDriver, WebDriverSession, bypass_cloudflare
+from ...typing import CreateResult, Messages
+from ..base_provider import AbstractProvider
+from ..helper import format_prompt
+from ...webdriver import WebDriver, WebDriverSession, bypass_cloudflare
```

Comment:
Avoid importing modules from parent directories, as it can lead to potential issues with module resolution.